### PR TITLE
Added a couple minor instructions to README.md

### PR DESCRIPTION
**Pull request by cstolli**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,6 +42,10 @@ the following dependencies as well: `graphicsmagick` and `cairo`
     sudo apt-get install graphicsmagick cairo
 
 ### Install `beaker`
+Before installing Beaker, make sure you have a `node_modules` directory in your project folder:
+
+    mkdir node_modules
+
```
Comment (job13er) [inline README.md]: Why is this needed? Whenever I bootstrap a new `beaker` project. I never manually create a `node_modules` directory. The `npm install beaker` will do that for you.

File: README.md [hunk 2]
```diff
@@ -72,6 +76,8 @@ If you are unfamiliar with Grunt you may want to head over to
 
 
 ### Verify
+After installing beaker, make sure to `npm install` in order to get the packages in the `package.json` placed by Beaker.
```
Comment (job13er) [inline README.md]: Actually, it should probably be `make install` since at least one of our current dependencies needs extra love after the `npm install` which is encapsulated in `make install`

File: README.md (hunk 1 above)
Comment (danjres) [inline README.md]: weird, whenever I create one, it fails unless I create the folder.

File: README.md (hunk 2 above)
Comment (danjres) [inline README.md]: ok that's better.

File: README.md (hunk 1 above)
Comment (job13er) [inline README.md]: So when you do an `npm install beaker` in an empty directory, it fails unless `node_modules/` exists already?
